specs/cluster_list_view.adoc

```diff
@@ -0,0 +1,140 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of clusters discovered by Tendrl
+
+Display list of Ceph and Gluster clusters present in Tendrl. The list should have functionality for 
+
+* filtering by name and provider.
+* sorting by name, status and utilisation.
+
+Cluster list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of Ceph and Gluster clusters present by Tendrl along with their details.
+
+The details shown for each cluster in UI is -
+
+* Cluster Name
+* Status
+* Storage Usage
+* CPU Utilisation
+* IOPS
+* Host Count
+* Pool Count
+* Alert Count
+
+API support present for -
+
+* Status
+* Host Count
+* Pool Count
+
+== Use Cases
+
+Users can see the list of cluster present in Tendrl. They can also 
+
+* filter by name and provider.
+* sort by name, status and utilisation.
+
+Users can't perfrom filter and sort operations because name, provider, status and utilisation data is not supported by API. 
+
+== Proposed change
+
+* Create a view for listing cluster details.
+* For showing storage usage and cpu utilisation, use donut chart component.
+* Implement client side filer and sorting functionality.
+* Filter call will trigger after 1 sec the user enters the filter data.
+* Send polling call after 10 mins to get the updated data. 
+* Total count of clusters will be displayed.
+
+=== Alternatives
+
+Generate list component can be created and used for listing the cluster. But because of its complexity and time constraint, we will pick this up later.
+
+=== Data model impact:
+
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+/api/1.0/GetClusterList
```

Comment:
This part lack information. state what is available currently(cluster wide list API available), what extra information reqd in the current API. Also specify the any extra APIs required(syatem wide listing?)
Which component implements sorting and filtering?
Any APis required to get the utlization data?